naples/oauth2.py

```diff
@@ -1,16 +1,17 @@
-from datetime import datetime, timedelta, UTC
 from fastapi import status
 from jose import JWTError, jwt
 from fastapi import HTTPException
 from pydantic import ValidationError
 
 import naples.schemas as s
+from naples.utils import get_expire_datatime
 from .config import config
 
 CFG = config()
 
 SECRET_KEY = CFG.JWT_SECRET
 ACCESS_TOKEN_EXPIRE_MINUTES = CFG.ACCESS_TOKEN_EXPIRE_MINUTES
+EXPIRE_DATETIME = get_expire_datatime()

```

Comment:
This is HUGE mistake, since this will create a fixed timestamp at the start of FASTAPI server, which will leads to users NOT being able to use authorised routes right after they logged in. Please fix this ASAP

Reply:
ok